doc/sphinx/proof-engine/tactics.rst

```diff
@@ -168,29 +169,22 @@ The most common splitting patterns are:
 
 * split a hypothesis in the form :n:`A /\ B` into two
   hypotheses :g:`H1: A` and :g:`H2: B` using the pattern :g:`(H1 & H2)` or
-  :g:`(H1, H2)` or :g:`[H1 H2]`
-  :ref:`Example <intropattern_conj_ex>`
+  :g:`(H1, H2)` or :g:`[H1 H2]`.
+  :ref:`Example <intropattern_conj_ex>`.  This also works on :n:`A <-> B`, which
+  is just a notation representing :n:`(A -> B) /\ (B -> A)`.
```

Comment:
Correct. But note that there were discussions of creating a proper inductive type for `<->` in which case this splitting pattern would still work. That's the magic :smile: